src/js/player.js

```diff
@@ -503,7 +506,7 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted,
+      'muted': this.muted_,
```

Comment:
Let's change this back to options.muted and remove the muted_ setting above. In the future we need to handle properties like this better. Decide where the state should live and how to pass that between techs.